tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -295,3 +298,69 @@ def test_positive_prism_central_prism_central_option(
         command = get_configure_command(virtwho_config.id, default_org.name)
         deploy_configure_by_command(command, form_data['hypervisor_type'], org=default_org.label)
         assert get_configure_option("prism_central", config_file) == 'true'
+
+    @pytest.mark.tier2
+    def test_positive_ahv_internal_debug_option(
+        self, default_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify ahv_internal_debug option by hammer virt-who-config"
+
+        :id: aa6a7443-62ae-4a87-8c27-4a7808f4b1da308
+
+        :expectedresults:
+            enable-ahv-debug option has been set to no
+            ahv_internal_debug bas been set to false in virt-who-config-X.conf
+            warning message exist in log file /var/log/rhsm/rhsm.log
+            ahv_internal_debug option can be updated
+            message Host UUID {system_uuid} found for VM: {guest_uuid} exist in rhsm.log
+            ahv_internal_debug bas been set to true in virt-who-config-X.conf
+            warning message does not exist in log file /var/log/rhsm/rhsm.log
```

Comment:
Numbering expectedresults would look nice
```suggestion
        :expectedresults:
            1. enable-ahv-debug option has been set to no
            2. ahv_internal_debug bas been set to false in virt-who-config-X.conf
            3. warning message exist in log file /var/log/rhsm/rhsm.log
            4. ahv_internal_debug option can be updated
            5. message Host UUID {system_uuid} found for VM: {guest_uuid} exist in rhsm.log
            6. ahv_internal_debug bas been set to true in virt-who-config-X.conf
            7. warning message does not exist in log file /var/log/rhsm/rhsm.log
```